tools/batch/template/launch-template-data-linux1(gpu).json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
it's bad to have parenthesis in the filename

Reply:
Because ECS will use linux1 as AMI for GPU related instance, and linux2 for CPU related instance. I feel it's necessary to mark it in the file name. If I do linux1-gpu, people might mistakenly think that there's a linux1-cpu too. Therefore, I put a parenthesis here. Is there any better naming convention I can use to achieve the same goal?